.coveragerc

```diff
@@ -0,0 +1,4 @@
+[run]
+branch = true
+source = dolphin
```

Comment:
Are there any third party files which we don't want to include? this will try to run coverage on whole dolphin. Should we use 'include' to be more specific? Or should use explicit 'omit'?

Reply:
currently there is no other file needs to be excluded.